src/status_im/ui/screens/browser/views.cljs

```diff
@@ -95,6 +96,12 @@
          :local-storage-enabled                 true
          :start-in-loading-state                true
          :render-error                          web-view-error
+         :on-error                              (fn [_]
+                                                  (utils/show-popup "Error"
+                                                                    (i18n/label :t/web-view-error)
+                                                                    #(do
+                                                                       (re-frame/dispatch [:remove-browser browser-id])
+                                                                       (re-frame/dispatch [:navigation-replace :open-dapp]))))
```

Comment:
why we can't just leave him on browser screen?

Reply:
@flexsurfer oh, should we leave him on browser screen?
if I understand correctly, @rachelhamlin wants the user returned to the "Open DApp" screen in the case of a bad search.

Reply:
1 - user can change url on browser page and 2 - browser can be opened from chat not only from "Open Dapp" screen

Reply:
so from my pov better to  leave him on browser screen, thanks , sorry for confusion

Reply:
I was referring to when the user hits the "back" arrow button on the error screen. Currently the user is taken back to Home screen.

It would be better if the "back" button brought the user to the previous screen, Open DApp. Not applicable if browser is opened from chat.